soh/src/overlays/actors/ovl_En_Fr/z_en_fr.c

```diff
@@ -979,6 +983,7 @@ void EnFr_SetReward(EnFr* this, PlayState* play) {
     } else if (songIndex == FROG_CHOIR_SONG) {
         if (!(gSaveContext.eventChkInf[13] & sSongIndex[songIndex])) {
             gSaveContext.eventChkInf[13] |= sSongIndex[songIndex];
+            GameInteractor_ExecuteOnFlagSet(FLAG_EVENT_CHECK_INF, (EVENTCHKINF_SONGS_FOR_FROGS_INDEX << 4) + (int)log2(sSongIndex[songIndex]));
```

Comment:
considering how few songs there are, we can probably avoid including math.h and doing an int cast here by just making a new array with all the needed values and looking up those by index